etc/pgbouncer.ini

```diff
@@ -266,6 +266,12 @@ auth_file = /etc/pgbouncer/userlist.txt
 ;; failure. (default: 120)
 ;query_wait_timeout = 120
 
+;; Dangerous.  Client connection is closed if the cancellation request
```

Comment:
I wonder if the "Dangerous" is really appropriate here, or was it just copied from the `query_` settings?

Reply:
Disabling it is definitely dangerous, but yeah it was mostly copied.